Firebase/Auth/Source/AuthProviders/Phone/FIRPhoneAuthProvider.m

```diff
@@ -364,6 +365,20 @@ - (void)internalVerifyPhoneNumber:(NSString *)phoneNumber
 - (void)verifyClientAndSendVerificationCodeToPhoneNumber:(NSString *)phoneNumber
                              retryOnInvalidAppCredential:(BOOL)retryOnInvalidAppCredential
                                                 callback:(FIRVerificationResultCallback)callback {
+  if (_auth.settings.isAppVerificationDisabledForTesting) {
+    FIRSendVerificationCodeRequest *request =
+        [[FIRSendVerificationCodeRequest alloc] initWithPhoneNumber:phoneNumber
+                                                     appCredential:nil
+                                                    reCAPTCHAToken:nil
+                                              requestConfiguration:
+                                                  self->_auth.requestConfiguration];
```

Comment:
Nit: Is there a reason you use `self->` here but not above in `_auth.settings. isAppVerificationDisabledForTesting`?

Reply:
Good catch, removed it to improve readability.